packages/dashboard/src/features/dashboard/components/observability/ObservabilitySection.tsx

```
@@ -186,16 +186,39 @@ export function ObservabilitySection() {
+      ? breakdown.database.map((point, i) => ({
+          timestamp: point.timestamp,
+          value: point.value + (breakdown.wal[i]?.value ?? 0),
+        }))
+      : diskUsedData;
     return {
-      data: diskUsedData,
-      fixedDomain: (totalBytes !== null ? [0, totalBytes] : undefined) as
+      data: dataUsed,
+      fixedDomain: (dataCeiling !== null ? [0, dataCeiling] : undefined) as
         | [number, number]
         | undefined,
```

ESLint

[error] Replace `|·[number,·number]⏎·······` with `[number,·number]`

(prettier/prettier)